RecFCCeeCalorimeter/src/components/CaloTowerToolFCCee.h

```diff
@@ -182,6 +182,17 @@ class CaloTowerToolFCCee : public AlgTool, virtual public ITowerToolThetaModule
   SegmentationType m_hcalFwdSegmentationType;
   /// decoder: only for barrel
   dd4hep::DDSegmentation::BitFieldCoder* m_decoder;
+<<<<<<< HEAD
+=======
+  // Set of bools to record whether a proper segmentation was found
+  bool m_ecalBarrelSegmentationOK;
+  bool m_ecalEndcapSegmentationOK;
+  bool m_ecalFwdSegmentationOK;
+  bool m_hcalBarrelSegmentationOK;
+  bool m_hcalExtBarrelSegmentationOK;
+  bool m_hcalEndcapSegmentationOK;
+  bool m_hcalFwdSegmentationOK;
```

Comment:
there was a problem with rebasing it seems, as some old code that was removed came back...